src/textual/widgets/_placeholder.py

```diff
@@ -147,6 +148,6 @@ def on_click(self) -> None:
 
     def on_resize(self, event: events.Resize) -> None:
         """Update the placeholder "size" variant with the new placeholder size."""
-        self._renderables["size"] = self._SIZE_RENDER_TEMPLATE.format(*self.size)
+        self._renderables["size"] = self._SIZE_RENDER_TEMPLATE.format(*event.size)
         if self.variant == "size":
-            self.refresh(layout=True)
+            self.refresh(layout=False)
```

Comment:
This is interesting! IIRC, when I tried implementing the placeholder, it wouldn't work properly if I didn't ask it to do a layout as well. I probably got it wrong. 😛

Reply:
Doesn't seem to be necessary now. Possibly it was a workaround for a bug which is now fixed.